template-manager-core/src/main/java/com/sequenceiq/cloudbreak/template/TemplateModelContextBuilder.java

```diff
@@ -65,11 +64,10 @@ public TemplateModelContextBuilder withGeneralClusterConfigs(GeneralClusterConfi
         return this;
     }
 
-    public TemplateModelContextBuilder withRdsConfigs(Iterable<RdsConfigWithoutCluster> rdsConfigs) {
-        for (RdsConfigWithoutCluster rdsConfig : rdsConfigs) {
-            if (rdsConfig != null) {
-                RdsView rdsView = new RdsView(rdsConfig);
-                String componentName = rdsConfig.getType().toLowerCase();
+    public TemplateModelContextBuilder withRdsConfigs(Iterable<RdsView> rdsViews) {
```

Comment:
Similarly to its `TemplatePreparationObject` counterpart, this could be also renamed to `withRdsViews()`. Leaving to you to decide; likely not worth the effort if there are many references.